ChangeLog.rst

```diff
@@ -56,6 +56,8 @@ deploy instead of falling back to the chart default (:ghpull:`197`)
 
 :ghissue:`220` - 'Kubernetes Calico (Alternative)' dashboard doesn't work (:ghpull:`221`)
 
+:ghissue:`321` - race condition in CI (:ghpull:`319`)
```

Comment:
Consider replacing this with the current PR title.

Reply:
We don't use the title of the issue for a bug fix ?